tests/benchmarks/app_kernel/src/master.h

```diff
@@ -23,7 +23,7 @@
 
 
 /* uncomment the define below to use floating point arithmetic */
-/* #define FLOAT */
+/* #define FPU */
```

Comment:
I don't think that this (and related changes) are correct. This is local define (`FLOAT`) of  tests/benchmarks/app_kernel, not directly related to Zephyr config option `CONFIG_FLOAT`. And it's meaning seems to be exactly use floating-point arithmetic (which can be also soft-float), rather than specifically FPU.

So, I'd suggest these changes to be reverted.

Reply:
Good catch, I will revert these.

Reply:
Reverted